selfdrive/ui/qt/home.cc

```diff
@@ -50,11 +50,22 @@ void HomeWindow::offroadTransition(bool offroad) {
   emit offroadTransitionSignal(offroad);
 }
 
+void HomeWindow::driverView() {
+  if (!driver_view) {
```

Comment:
think it makes more sense for driver view to live in the settings window or maybe main window

Reply:
why lazy init? is it slow?